source/Playnite.DesktopApp/DesktopApplication.cs

```diff
@@ -230,5 +230,13 @@ private bool ProcessStartupWizard()
 
             return isFirstStart;
         }
+
+        public override void NotifyInWindows(string title, string body)
+        {
+            var icon = new Icon(ThemeFile.GetFilePath("Images/applogo.ico", ThemeManager.DefaultTheme,
+                ThemeManager.CurrentTheme));
+            trayIcon.ShowBalloonTip(title, body, icon, true);

```

Comment:
This will throw null reference exception if tray icon is disabled in settings.

Reply:
resolved in the recent commits below